### ISSUE TITLE
Please add section to be able to comment or ask question in documentation pages

### ISSUE THREAD

**Issue by RezaRahmati** [nestjs/nest#2626]
## Is your feature request related to a problem? Please describe.

As a user of nestjs I would like to be able to ask question or give feedback about documentations in the page I view the content.

## Describe the solution you'd like

Microsoft recently added this feature, see this [https://docs.microsoft.com/en-us/azure/container-instances/container-instances-volume-azure-files](example) , in the bottom of page if you click on "Send feed back about this page", then you can give your feedback and an item will be created on their github page and it's very cool.  

## What is the motivation / use case for changing the behavior?

I read the Logging section of nestjs document, and find out there is nothing about how to use it, after googling find out the others also having same issue with that page, so why not to give feedback or even participate in completing the documentation, and better to be able to track them via github issues

**Comment by dantehemerson** [comment]
I think this issue should go in the [docs](https://github.com/nestjs/docs.nestjs.com) repository

**Comment by RezaRahmati** [comment]
@dantehemerson Yes, right, How can I move it there? or should I resubmit it in that repo?

**Comment by dantehemerson** [comment]
@RezaRahmati I think some maintainer could move it.
@kamilmysliwiec  You could move this to the docs repository?.
